fields-flow/build.gradle.kts

```diff
@@ -10,6 +10,8 @@ plugins {
 }
 
 dependencies {
+    androidMainApi(libs.androidXLifecycle)
+    androidMainApi(libs.lifecycleKtx)
```

Comment:
it's already available as api in `fieldsCore`

Reply:
My mistake, removed